src/components/User/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React,{useContext} from "react";
+import { StyledButton } from "./styles";
+import { useHistory } from "react-router-dom";
+ import {UserContext} from "../../UserContext";
+
+const User = () => {
+  const {login , password } = useContext(UserContext);
+
+  const history = useHistory();
+
+  const logIn = () => {    
+    history.push('/')
```

Comment:
here you should remove token from context which should cause to redirect onto login page